src/utils.js

```diff
@@ -32,4 +32,63 @@ H5P.Components.utils.createElement = (tag, options, style = {}) => {
   Object.assign(element.style, style);
 
   return element;
-};
\ No newline at end of file
+};
+
+/** @constant {number} DEFAULT_LINE_HEIGHT Default line height when it is "normal" */ 
+const DEFAULT_LINE_HEIGHT = 1.2;
+/**
+ * Compute the number of lines in an element.
+ * @param {HTMLElement} element The element to compute lines for.
+ * @returns {number} The number of lines in the element.
+ */
+H5P.Components.utils.computeLineCount = (element) => {
+  if (!element) {
+      return 0; 
```

Comment:
Wrong indentation and a trailing space. I know this is linter business, but if no linter is used, we need to take care of this.